libs/utils/analysis/frequency_analysis.py

```diff
@@ -133,13 +133,13 @@ def plotClusterFrequencies(self, title='Clusters Frequencies'):
 
         # Extract LITTLE and big clusters frequencies
         # and scale them to [MHz]
-        if len(self._platform['clusters']['little']):
-            lfreq = df[df.cpu == self._platform['clusters']['little'][-1]]
+        if self._little_cpus:
+            lfreq = df[df.cpu == self._little_cpus[-1]]
             lfreq['frequency'] = lfreq['frequency']/1e3
         else:
             lfreq = []
```

Comment:
In the if branch, `lfreq` is a dataframe while here is set to an empty list. Sholdn't we set it to an empty dataframe? Just to avoid confusion...

Reply:
Maybe, but that's nothing to do with this PR. AFAICT it doesn't affect the return value/plot output.